extensions-contrib/opentsdb-emitter/src/main/java/org/apache/druid/emitter/opentsdb/EventConverter.java

```diff
@@ -54,9 +55,14 @@ protected String sanitize(String metric)
     return WHITESPACE.matcher(metric.trim()).replaceAll("_").replace('/', '.');
   }
 
-  private String buildNamespace()
+  private String buildMetric(String metric)
   {
-    return (namespacePrefix == null || "".equals(namespacePrefix)) ? "" : sanitize(namespacePrefix) + ".";
+    final String sanitized = sanitize(metric);
+    if (Strings.isNullOrEmpty(namespacePrefix)) {
```

Comment:
I think this can just check `namespacePrefix == null` now

Reply:
Fixed in latest commit